### DISCUSSION TITLE
Issue with Drag and Drop Functionality on Safari Browser Grapesjs

### DISCUSSION THREAD

**Discussion by riteshagarwalsw** [GrapesJS/grapesjs#5856] [Q&A]
@artf  I wanted to bring up an issue regarding the drag and drop functionality within the canvas on the **Safari browser**. It seems that when dragging using the toolbar icon, the functionality doesn't seem to work as expected. I'm wondering if this is a known limitation or if there's a solution available.

You can reproduce the issue using the following link: [GrapesJS Demo](https://grapesjs.com/demo.html)

Any insights or guidance on how to address this would be greatly appreciated. Thank you for your attention to this matter.

**Comment by artf** [comment]
Which version of Safari?

**Reply by riteshagarwalsw** [reply]
I am using Safari version 17.4.1
<img width="284" alt="Screenshot 2024-05-07 at 7 56 45 AM" src="https://github.com/GrapesJS/grapesjs/assets/79893029/73732662-da1e-4c85-9a6a-34f2f6d048b6">

**Reply by riteshagarwalsw** [reply]
Attaching a recording of the issue.

https://github.com/GrapesJS/grapesjs/assets/79893029/7cad6d5c-69f1-413f-80a8-8934ce161875

**Reply by artf** [reply]
weird, any error in console?

**Reply by riteshagarwalsw** [reply]
No, there are no errors in the console. I tried to debug a lot and I think there is some issue in the startSort function.
`/**
   * Picking component to move
   * @param {HTMLElement} src
   * */
  startSort: function startSort(src) {
   // Other lines
   Object(utils_mixins__WEBPACK_IMPORTED_MODULE_3__["on"])(container, 'mousemove dragover', this.onMove);
}`
In above code the onMove function never gets triggered. It's seems container is unable to bind the function in Safari.

**Reply by riteshagarwalsw** [reply]
Attaching recording with console

https://github.com/GrapesJS/grapesjs/assets/79893029/25e36c66-9be6-4a24-a111-2a9bdd7468d7
